.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
+
+[*]
+charset = utf-8
```

Comment:
So because of this the PR is also removing the utf-8 BOM in all files, which we should at least mention in the commit message?

Reply:
Is this also something that is currently inconsistent, or should we use `utf-8-bom` here to minimize changes?